### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by l1ttleO**

### CODE REVIEW

File: resources/plugin.yml [hunk 1]
```diff
@@ -13,12 +13,12 @@ description: Provides on-the-fly system for permissions system created by Nijiko
...
 permissions:
     groupmanager.*:
-        description: Wildcard permission to grant all permissions.
+        description: A wildcard to grant all permissions.
         default: op
         children:
             groupmanager.op: true
     groupmanager.op:
-        description: User is treated as an op when using the GroupManager commands.
+        description: Given if the user is a server operator
```
Comment (ElgarL) [inline resources/plugin.yml:21]: This is wrong. This permission when granted will allow the user control over GM as if he were OP.
It is granted to ops but thats not the purpose of the node.

File: src/languages/english.properties [hunk 2]
```diff
@@ -27,8 +27,8 @@ ERROR_USER_NO_ACCESS_VARIABLE_DIRECT=The user doesn't have direct access to that
 
 VARIABLE_REMOVED_FROM_USER=Variable %s removed from the user %s.
 VARIABLE_REMOVED_FROM_GROUP=Variable %s removed from the group %s.
-VARIABLE_ADDED_TO_USER=Variable %s:'%s' added to the user %s.
-VARIABLE_ADDED_TO_GROUP=Variable %s:'%s' added to the group %s.
+VARIABLE_ADDED_TO_USER=Variable %s: '%s' added to the user %s.
+VARIABLE_ADDED_TO_GROUP=Variable %s: '%s' added to the group %s.
```
Comment (ElgarL) [inline src/languages/english.properties:31]: These are intentional formatting spaces.

File: src/languages/english.properties [hunk 3]
```diff
@@ -45,15 +45,15 @@ GROUP_HAS_PERMISSIONS=The group '%s' has the following permissions: %s
 USER_HAS_PERMISSIONS=The player '%s' has the following permissions: %s
 GROUP_NO_SPECIFIC_PERMISSIONS=The group '%s' has no direct permissions.
 USER_NO_SPECIFIC_PERMISSIONS=The player '%s' has no direct permissions.
-ERROR_SAME_PERMISSIONS_OR_HIGHER=You can't modify a player the with same permissions as you, or higher.
+ERROR_SAME_PERMISSIONS_OR_HIGHER=You can't modify a player with same permissions as you, or higher.
```
Comment (ElgarL) [inline src/languages/english.properties:48]: You can't modify a player with the same permissions as you, or higher

File: src/languages/english.properties [hunk 4]
```diff
@@ -45,15 +45,15 @@ GROUP_HAS_PERMISSIONS=The group '%s' has the following permissions: %s
...
 ERROR_SAME_GROUP_OR_HIGHER=You can't modify a player with the same group as you, or higher.
 ERROR_DIFFERENT_INHERITANCE_LINES=You can't modify a player using groups with different heritage lines.
 ERROR_DESTINATION_HIGHER_THAN_YOURS=The destination group can't be the same as yours, nor higher.
 ERROR_SUBGROUP_HIGHER_THAN_YOURS=The sub-group can't be the same as yours, nor higher.
 ERROR_NEW_GROUP_NOT_HIGHER=The new group must be a higher rank than the current one.
 ERROR_NEW_GROUP_NOT_LOWER=The new group must be a lower rank than the current one.
-ERROR_PLAYERS_NOT_MEMBERS_OF_GG=Players may not be members of GlobalGroups directly.
-ERROR_YOU_DO_NOT_INHERIT=You can't modify a player involving a group that you don't inherit.
+ERROR_PLAYERS_NOT_MEMBERS_OF_GG=Players cannot be members of GlobalGroups directly.
+ERROR_YOU_DO_NOT_INHERIT=You can't modify a player that inherits a group that you don't inherit.
```
Comment (ElgarL) [inline src/languages/english.properties:56]: The original here is correct. Its only in relation to the group in question.

File: src/languages/english.properties [hunk 5]
```diff
@@ -97,15 +97,15 @@ RELOAD_REQUEST_ATTEMPT=The request to reload world '%s' was attempted.
 RELOADED=All settings and worlds were reloaded\!
 
 AUTOSAVE_ENABLE=Auto-saving is enabled\!
-AUTOSAVE_DISABLE=Auto-saving is disabled\! Timed permissions will not expire while saving is disabled\!
+AUTOSAVE_DISABLE=Auto-saving is disabled\! Timed permissions and subgroups will not expire while saving is disabled\!
 VALIDATE_STATUS=Validate if a player is online, now set to: 
 VALIDATE_1=From now on you can edit offline players... BUT:
-VALIDATE_2=You should type the whole name of the player, correctly.
+VALIDATE_2=Correctly, you should type the whole name of the player.
```
Comment (ElgarL) [inline src/languages/english.properties:103]: This sounds confusing.

File: src/org/anjocaido/groupmanager/GlobalGroups.java [hunk 6]
```diff
@@ -408,9 +408,8 @@ public PermissionCheckResult checkPermission(String groupName, String permission
 	}
 
 	/**
-	 * Returns a Set of all global group names.
 	 * 
-	 * @return Set containing all group names.
+	 * @return a set containing all group names.
```
Comment (ElgarL) [inline src/org/anjocaido/groupmanager/GlobalGroups.java:412]: Capitalize Set as its a java object type.

File: src/org/anjocaido/groupmanager/GroupManager.java [hunk 7]
```diff
@@ -617,9 +617,9 @@ public static GlobalGroups getGlobalGroups() {
 	}
 	
 	/**
-	 * Is the plug-in fully loaded?
+	 * Is the plugin fully loaded?
 	 * 
-	 * @return	true if we are loaded or false if reloading/loading.
+	 * @return true if we are loaded or false if reloading/loading.
```
Comment (ElgarL) [inline src/org/anjocaido/groupmanager/GroupManager.java:622]: Tabs were correct to align in code

**Review by ElgarL — CHANGES_REQUESTED** [review]

File: src/languages/english.properties (hunk 2 above)
Comment (l1ttleO) [reply src/languages/english.properties:31]: What do you mean? It looks nicer to have a space after `:`

File: src/languages/english.properties (hunk 3 above)
Comment (l1ttleO) [reply src/languages/english.properties:48]: In my opinion, what I already have is correct, but I will change to what you say anyway

File: src/languages/english.properties (hunk 2 above)
Comment (ElgarL) [reply src/languages/english.properties:31]: Yes, its clearer in the final message

**Review by ElgarL — APPROVED** [review]

**Pull request merged**